modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/hooks/useIframeClient.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {useCallback, useEffect, useRef, useState} from 'react';
+
+import {SDK} from '../SDK';
+
+export enum Kind {
+	Click = 'a11y:popover:click',
+}
```

Comment:
Since this enum has just one element, why not hard code the string in `Kind.Click` usages or create just a string const called `KindClick` or something like it? Do you have plans to add more interactions in this `Kind`?

Reply:
We don't have plans to add more interactions but it's likely that we will have more in the future, this is more of a pattern for the term `Kind`, we can't use `type` here as `PopoverClick` because we can't use it as a constant unlike `enum` which is made for just that.